root/annotation/AnnotationComparison.js

```diff
@@ -50,17 +50,17 @@ const AnnotationComparison = ({
           <p>
             <DiffSide
               filter={DELETE}
-              newText={newAnnotation.text}
-              oldText={oldAnnotation.text}
+              newText={newAnnotation.text ?? ''}
+              oldText={oldAnnotation.text ?? ''}
               split="\s+"
             />
           </p>
           <h3>{l('New annotation')}</h3>
           <p>
             <DiffSide
               filter={INSERT}
-              newText={newAnnotation.text}
-              oldText={oldAnnotation.text}
+              newText={newAnnotation.text ?? ''}
+              oldText={oldAnnotation.text ?? ''}
```

Comment:
The displayed diff is not the same as it currently is in production. Why? Is it intended? See https://musicbrainz.org/release/225f760b-f8d8-47a3-b74c-a3ca1a0c0e96/annotation/539363 for example.

Reply:
It is, it's the same as it would be currently in beta :) See https://beta.musicbrainz.org/release/225f760b-f8d8-47a3-b74c-a3ca1a0c0e96/annotations-differences?new=539363&old=276825

The reasoning is in https://github.com/metabrainz/musicbrainz-server/pull/2491/commits/1002579308b4a1699331f3956492833e26f59fff